### ISSUE TITLE
Wrong calculations

### ISSUE THREAD

**Issue by simoheinonen** [chriskonnertz/string-calc#1]
```php
$calculator = new ChrisKonnertz\StringCalc\StringCalc();
echo $calculator->calculate('1-1-1') . PHP_EOL;
```
prints `1`
 ```php
$calculator = new ChrisKonnertz\StringCalc\StringCalc();
echo $calculator->calculate('1-1+1') . PHP_EOL;
```
prints `-1`

**Comment by chriskonnertz** [comment]
Hello,

I will check this tomorrow.

**Comment by chriskonnertz** [comment]
I've added your example calculations to the test calculations: https://github.com/chriskonnertz/string-calc/commit/50c75e9b193b9a5b204bf0e81226427b5cbe0731

I will investigate why they cause wrong results.

**Comment by chriskonnertz** [comment]
Seem's to be an issue with PHP 5.6's implementation of Quicksort (uasort). It seems to put items with the same comparison value (0) in the opposite order as PHP >= 7. I think I worked with HP 7.0 here so I did not notice that.

**Comment by chriskonnertz** [comment]
Bug has been fixed for PHP 5.6: https://github.com/chriskonnertz/string-calc/commit/f131daa587e77ad2f0689fc980f82aa49a12b093
Test is green now: https://travis-ci.org/chriskonnertz/string-calc/jobs/260698794
Sorry for the inconvenience.
